qiskit/transpiler/passes/calibration/pulse_gate.py

```diff
@@ -97,4 +97,4 @@ def get_calibration(self, node_op: CircuitInst, qubits: List) -> Union[Schedule,
         Raises:
             TranspilerError: When node is parameterized and calibration is raw schedule object.
         """
-        return self.target._get_calibration(node_op.name, tuple(qubits), *node_op.params)
+        return self.target.get_calibration(node_op.name, tuple(qubits), *node_op.params)
```

Comment:
```suggestion
        return self.target._get_calibration(node_op.name, tuple(qubits), *node_op.params)
```
We are calling here the internal method purposely to avoid getting pulse deprecation warnings when calling `PulseGate.get_calibration()` itself. The entire class is deprecated anyway so no need to get the internal warnings out of this method